templates/main-nav.html

```diff
@@ -95,7 +95,7 @@
         <li><a href="/desktop/whitepapers">Whitepapers</a></li>
       </ul>
     </li>
-    <li class="p-navigation__link">
+    <li class="p-navigation__link{%- if page_slug == 'topics' %} is-selected{% endif %}">
       <a href="#">Topics</a>
```

Comment:
Not really related to this PR, but 'Topics' is the only item in navigation that cannot be clicked which is a bit confusing.